src/core/Akka/Event/StandardOutLogger.cs

```diff
@@ -115,6 +101,11 @@ protected override void Log(object message)
         /// </summary>
         public static bool UseColors { get; set; }
 
+        /// <summary>
+        /// FOR TESTING PURPOSES ONLY, DO NOT USE
+        /// </summary>
+        internal event Action<object> Invoked;
```

Comment:
This is the only addition to the code, I have to add this in for testing because there is no other way to check if StandardOutLogger was called or not.